lib/convert.js

```diff
@@ -859,6 +863,10 @@ module.exports = function convert(config) {
                 result.key.optional = true;
             }
 
+            if (node.exclamationToken) {
+                result.exclamationToken = true;
```

Comment:
I [suggested](https://github.com/babel/babel/pull/7159#discussion_r167437266) `definite` in the Babylon PR, what do you think?

Reply:
We'll follow babylon's choice - updated to `definite`.

Reply:
Thanks 👍